packages/js/src/utils/id-token.ts

```diff
@@ -0,0 +1,68 @@
+import * as s from 'superstruct';
+
+const fullfillBase64 = (input: string) => {
+  if (input.length === 2) {
+    return `${input}==`;
+  }
+
+  if (input.length === 3) {
+    return `${input}=`;
+  }
+
+  return input;
+};
+
+/**
+ * @link [ID Token](https://openid.net/specs/openid-connect-core-1_0.html#IDToken)
+ */
+const IdTokenClaimsSchema = s.type({
+  iss: s.string(),
+  sub: s.string(),
+  aud: s.string(),
+  exp: s.number(),
+  iat: s.number(),
+  auth_time: s.optional(s.number()),
+  nonce: s.optional(s.string()),
+  acr: s.optional(s.string()),
+  amr: s.optional(s.array(s.string())),
+  azp: s.optional(s.string()),
+  at_hash: s.optional(s.string()),
+  c_hash: s.optional(s.string()),
```

Comment:
please follow our sdk convention to reflect the true state of Logto. those optional props can be added later

Reply:
Removed.